services/provider/server/server.go

```diff
@@ -427,6 +427,17 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 
 	}
 
+	subscriptionSpec, err := s.getOCSSubscriptionSpec(ctx)
```

Comment:
Why are we getting the ocs-operator subscription when we need to send the subscription for the client?

Reply:
This is the existing code, and that's what they are sending in the heartbeat. I have used the same function. I guess ocs-operator is the base for it, and that's the reason they are sending it in the existing code.